fixtures/contest/irled64-18x12six.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "irLED64 18x12SIX",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["Edrig"],
+    "createDate": "2023-04-20",
+    "lastModifyDate": "2023-04-20"
+  },
+  "links": {
+    "manual": [
+      "https://www.contest-lighting.com/home/downloads/IrLED64-18x12SIX_manuel.pdf"
```

Comment:
Link for the manual is not in English
Link for the English manual: https://www.contest-lighting.com/downloads/IrLED64-18x12SIX_manuel-V1-en.pdf